site/app/views/calendar/CalendarView.php

```diff
@@ -98,6 +98,8 @@ public function showCalendar(CalendarInfo $info, array $courses, bool $in_course
             "colors" => $info->getColors(),
             "instructor_courses" => $this->core->getQueries()->getInstructorLevelUnarchivedCourses($this->core->getUser()->getId()),
             "view_cookie" => isset($_COOKIE['view']) ? $_COOKIE['view'] : "month",
+            "month_cookie" => isset($_COOKIE['calendar_month']) ? $_COOKIE['calendar_month'] : $month, //cookie that can be accessed by twig file
```

Comment:
This is a good spot to use PHP's [null coalescing operator](https://www.tutorialspoint.com/php7/php7_coalescing_operator.htm)